README.md

```diff
@@ -1,58 +1,144 @@
+
+To connect and find support, join the multiplayer community on the Unity [Discord](https://discord.gg/mNgM2XRDpb) and [Forum](https://forum.unity.com/forums/multiplayer.26/).<br> Alternatively, tell us what you think using our new [Feedback Form](#feedback-form) <center>.
 | -- |
 
-## Table of content
+
+
+## Contents and Quick Links
 
 <!-- TOC generated from https://luciopaiva.com/markdown-toc/ -->
 
 
-- [Boss Room](#boss-room)
-- [Index of resources in this project](#index-of-resources-in-this-project)
-  - [Gameplay](#gameplay)
-  - [Game Flow](#game-flow)
-  - [Connectivity](#connectivity)
-  - [Services (Lobby, Relay, etc)](#services-lobby-relay-etc)
-  - [Tools and Utilities](#tools-and-utilities)
+- [Boss Room Overview](#boss-room-overview)
 - [Getting the project](#getting-the-project)
   - [Direct download](#direct-download)
   - [Installing Git LFS to clone locally](#installing-git-lfs-to-clone-locally)
 - [Registering the project with Unity Gaming Services (UGS)](#registering-the-project-with-unity-gaming-services-ugs)
 - [Opening the project for the first time](#opening-the-project-for-the-first-time)
-- [Testing multiplayer](#testing-multiplayer)
 - [Exploring the project](#exploring-the-project)
+- [Testing multiplayer](#testing-multiplayer)
+- [Index of resources in this project](#index-of-resources-in-this-project)
+  - [Gameplay](#gameplay)
+  - [Game Flow](#game-flow)
+  - [Connectivity](#connectivity)
+  - [Services (Lobby, Relay, etc)](#services-lobby-relay-etc)
+  - [Tools and Utilities](#tools-and-utilities)
+- [Troubleshooting](#troubleshooting)
+- [Licence](#licence)
 - [Other samples](#other-samples)
   - [Bite-size Samples](#bite-size-samples)
 - [Contributing](#contributing)
 - [Feedback Form](#feedback-form)
 
-## Boss Room
+## Boss Room Overview
 
-Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is built to serve as an educational sample that showcases certain typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom-examples/bossroom-actions) that are frequently featured in similar networked games.
+Boss Room is designed to be used in its entirety to help you explore the concepts and patterns behind a multiplayer game flow; such as character abilities, casting animations to hide latency, replicated objects, RPCs, and integration with the [Relay](https://unity.com/products/relay), [Lobby](https://unity.com/products/lobby), and [Authentication](https://unity.com/products/authentication) services.
 
-You can use everything in this project as a starting point or as bits and pieces in your own Unity games. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.
+You can use the project as a starting point for your own Unity game or use elements individually. 
```

Comment:
```suggestion
You can use the project as a reference starting point for your own Unity game or use elements individually. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.
```
mmm I'd really make it clear this is UCL (and not MIT for example). UCL has some restrictions. I also have a vague memory of legal telling us licences should be referenced in a "reasonably easy place to find" for legal reasons. --> readme was that.
I also changed the way this was worded, I wouldn't reused things in boss room as is, I'd use boss room as a reference. Boss Room wasn't made to be copy-pasted without customization.